primitives/historical-data/README.md

```diff
@@ -0,0 +1,18 @@
+## Historical data
+
+Crate with functionality to manage data that stores its own history.
+
+This covers:
+- linear history driven data, eg. transactional layers for overlay.
+- long term storage with multiple branch, eg. offchain storage.
```

Comment:
```suggestion
- long term storage with multiple branches, eg. off-chain storage.
```

Reply:
is that even the intended meaning?

Reply:
I am removing this line, it is something that is implemented in another PR, but since the code is currently not in the crate it does not make sense to keep it.